### PR TITLE
#6: insert not found Page (closes #6)

### LINKED ISSUE #6: insert not found Page
## Requirements

We need to add a 404 not found page, in order to give feedback for non existing resources
## Specs
- Build new route
- Use NotFoundRoute

### PR DESCRIPTION
**Pull request by nerder**
Issue #6
## Test plan
- Browse to /details/1 (should be ok)
- Browse to /foobar/1 (should return 404 page)

### CODE REVIEW

File: src/components/NotFound.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { skinnable } from 'revenge';
+
+@skinnable()
+
```
Comment (giogonzo) [inline src/components/NotFound.js]: stylistic comment: this should go right on top of the class declaration (no empty line in between)

File: src/routes/NotFoundRoute.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import NotFound from 'components/NotFound';
+
+export default class NotFoundRoute extends React.Component {
+
+  render() {
+    return (
+      <NotFound/>
+      );
```
Comment (giogonzo) [inline src/routes/NotFoundRoute.js:9]: one more: the closing paren should be aligned with `return`

**Comment by giogonzo** [comment]
looks good except for minor stylistic changes, feel free to merge yourself after you fixed those :+1:

**Comment by nerder** [comment]
@giogonzo this PR can be merged?

EDIT: didn't see that comment. I'm going to merge it

**Comment by giogonzo** [comment]
Yes, there's still a comment I left you didn't take care of, but very minor anyway, so merging

**Pull request merged**